examples/jobs/passjob.py

```diff
@@ -4,7 +4,29 @@
 
 from avocado.core.job import Job
 
-config = {'run.references': ['examples/tests/passtest.py:PassTest.test']}
+# from avocado.core.suite import TestSuite
 
-with Job(config) as j:
-    sys.exit(j.run())
+job_config = {'run.test_runner': 'nrunner',
+              'run.results_dir': '/tmp/foo42'}
+
+config1 = {'run.references': ['examples/tests/passtest.py:PassTest.test']}
+config2 = {'run.references': ['examples/tests/passtest.py:PassTest.test']}
```

Comment:
I would prefer to see a `passjob.py` example with the simplest configuration, one test suite. Then have a `passjob_with_suites.py` example covering the cases with more than one test suite.

Reply:
Ok, I'm splitting in two files for v2.